BismarkUnchained/scenes/Player/Player.gd

```diff
@@ -21,42 +19,26 @@ func damage():
 	get_node('/root/Main/UI/Health').value = health
 
 func move_translate(delta):
-	var velocity
-	
-	#Controller input
-	if controller:
-		velocity = Vector2(Input.get_joy_axis(device, 0), Input.get_joy_axis(device, 1))
-		if (velocity.length() < deadzone):
-			velocity = Vector2(0, 0)
-		if (velocity.length() > 1):
-			velocity = velocity.normalized()
-		#Cube curving to make small adjustments easier
-		velocity = velocity * velocity * velocity
-	
-	#If no controller input, use keyboard
-	else:
-		velocity = Vector2(0, 0)
-		if Input.is_action_pressed("player_up"):
-			velocity += Vector2(0, -1)
-		if Input.is_action_pressed("player_down"):
-			velocity += Vector2(0, 1)
-		if Input.is_action_pressed("player_left"):
-			velocity += Vector2(-1, 0)
-		if Input.is_action_pressed("player_right"):
-			velocity += Vector2(1, 0)
+	var velocity = Vector2(0, 0)
+	if Input.is_action_pressed("player_up"):
+		velocity += Vector2(0, -1)
+	if Input.is_action_pressed("player_down"):
+		velocity += Vector2(0, 1)
+	if Input.is_action_pressed("player_left"):
+		velocity += Vector2(-1, 0)
+	if Input.is_action_pressed("player_right"):
+		velocity += Vector2(1, 0)
 	
 	#Finding difference between current and target velocity, and adjusting for acceleration
-	velocity = velocity * max_velocity
+	velocity = velocity.normalized() * max_velocity
+	if Input.is_action_pressed("player_slowdown"):
+		velocity = velocity.normalized() * slow_velocity
```

Comment:
not sure if this affects the math, but you're normalizing twice when it makes sense to compute the velocity scalar (`max_velocity` or `max_velocity * slow_velocity`) then multiply by the `velocity.normalized()`